src/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
```

Comment:
why do we need to reset margin like this? 
Use `*` selector only for `box-sizing` property
reset margin directly on `body` element

Reply:
so used to use

Reply:
Add change